CodeStarterCamp_Week2/main.swift

```diff
@@ -8,23 +8,39 @@
 
 import Foundation
 
-let myLottoNumbers: Set<Int> =  [1, 2, 3, 4, 5, 6]
-var numberfull: [Int] = Array<Int>(1...45)
-var lotto: Set<Int> = Set<Int>()
+var lottoDictionary = [String : Set<Int>]()
+var numberFull: [Int] = Array<Int>(1...45)
+var lottoNumber: Set<Int> = Set<Int>()
```

Comment:
`Set<Int>` 타입으로 저장한다면 출력을 할 때마다 매번 같은 값이여도 다른 순서로 요소들이 출력될 것 같아요!
사용자에게 일관되게 출력결과를 주는것도 좋은 코드, 좋은 프로그램을 작성하는 방법일 것 같아요.
어떻게 수정해줄 수 있을까요??

Reply:
```suggestion
var lottoDictionary = [String : Array<Int>]()
var numberFull: [Int] = Array<Int>(1...45)
var lottoNumber: Set<Int> = Set<Int>()

func makeRandomLottoNumber() -> Array<Int> {
    var lottoNumber: Array<Int> = [Int]()
    while lottoNumber.count < 6 {
        lottoNumber.insert(Int.random(in: 1...45))
    }
    return lottoNumber
```
요청주신대로 타입을 일정하고 순서가 있는 Array로 수정해봤는데 컴파일에러가납니다. 그에 대한 구글링을 해봤지만 답을 찾기 어려워 재문의드립니다.

Reply:
`insert`메서드가 Array에도 있지만 지금 사용하신 형태는 Set의 `insert`메서드 인 것 같습니다.

```swift
func makeRandomLottoNumber() -> Array<Int> {
    var lottoNumber: Set<Int> = []
    while lottoNumber.count < 6 {
        lottoNumber.insert(Int.random(in: 1...45))
    }
    return Array(lottoNumber).sorted()
}
```

이렇게 사용하시면 마지막에 정렬된 값으로 반환해주기 때문에 매 출력마다 오름차순으로 이쁘게 출력해 줄 수 있을것 같네요!